ui.config/src/main/content/jcr_root/apps/acm-config/osgiconfig/config/org.apache.sling.jcr.repoinit.RepositoryInitializer~acm.config

```diff
@@ -8,9 +8,14 @@ scripts=["
     create path /conf/acm/settings/script/auto/enabled(sling:OrderedFolder)
     create path /var/acm(sling:OrderedFolder)
 
-    create service user acm-service with path system/acm/acm-service
-
-    set ACL for acm-service
+    create service user acm-content-service with path system/acm/acm-content-service
+    set ACL for acm-content-service
         allow jcr:all on /
     end
+
+    create service user acm-mock-service with path system/acm/acm-mock-service
```

Comment:
hard to say if this is accurate ; it's fail-safe default but security concerns still need to be resolved case by case by projects using the ACM tool